programs/nft-voter/src/instructions/relinquish_nft_vote.rs

```diff
@@ -0,0 +1,112 @@
+use anchor_lang::prelude::*;
+use spl_governance::state::{enums::ProposalState, governance, proposal};
+use spl_governance_tools::account::dispose_account;
+use crate::error::NftVoterError;
+use crate::tools::governance::get_vote_record_address;
+use crate::{state::*};
+
+use crate::state::{get_nft_vote_record_data_for_proposal_and_token_owner, Registrar};
+
+/// Disposes NftVoteRecord and recovers the rent from the accounts   
+/// It can only be executed when voting on the target Proposal ended or voter withdrew vote from the Proposal
+#[derive(Accounts)]
+pub struct RelinquishNftVote<'info> {
+    /// The NFT voting Registrar
+    pub registrar: Account<'info, Registrar>,
+
+    #[account(
+        mut,
+        constraint = voter_weight_record.realm == registrar.realm 
+        @ NftVoterError::InvalidVoterWeightRecordRealm,
+
+        constraint = voter_weight_record.governing_token_mint == registrar.governing_token_mint
+        @ NftVoterError::InvalidVoterWeightRecordMint,
+
+        constraint = voter_weight_record.governing_token_owner == governing_token_owner.key()
+        @ NftVoterError::InvalidVoterWeightRecordOwner,
+    )]
+    pub voter_weight_record: Account<'info, VoterWeightRecord>,
+
+    /// CHECK: Owned by spl-governance instance specified in registrar.governance_program_id
+    pub governance: UncheckedAccount<'info>,
```

Comment:
called `governance_program_id` elsewhere

Reply:
it's the `Governance` account and not `program_id`. Added comment to disambiguate